examples/vision/md/cait.md

```diff
@@ -143,9 +150,9 @@ class StochasticDepth(layers.Layer):
     def call(self, x, training=False):
         if training:
             keep_prob = 1 - self.drop_prob
-            shape = (tf.shape(x)[0],) + (1,) * (len(tf.shape(x)) - 1)
-            random_tensor = keep_prob + tf.random.uniform(shape, 0, 1)
-            random_tensor = tf.floor(random_tensor)
+            shape = (ops.shape(x)[0],) + (1,) * (len(x.shape) - 1)
+            random_tensor = keep_prob + ops.random.uniform(shape, 0, 1)
```

Comment:
Please add `seed=self.seed_generator` and create `self.seed_generator` in the constructor. It's a best practice

Reply:
Added, thanks for the review!